src-tauri/exploits/secretsdump.py

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/env python
```

Comment:
hmmm. can we perhaps add impacket as a submodule or and clone it in our install dependencies script

Reply:
apparently this now comes with kali

Reply:
made it a dep via 0177fdb

Reply:
Done 8453cfe